appserver/ejb/ejb-http-remoting/client/src/main/java/fish/payara/ejb/rest/client/RemoteEJBContextFactory.java

```diff
@@ -49,24 +49,14 @@
  * This is the context factory that creates the context used for looking up and invoking
  * remote EJBs.
  * 
- * <p>
- * Clients wanting to use remote EJB in this way should set the property in the initial context
- * environment as follows:
- * 
- * <pre>
- * <code>
- *   Hashtable&lt;String, String&gt; environment = new Hashtable&lt;&gt;();
- *   environment.put(INITIAL_CONTEXT_FACTORY, "fish.payara.ejb.rest.client.RemoteEJBContextFactory");
- *    ...
- *   new InitialContext(environment);
- * <code>
- * </pre>
- * 
+ * <p><strong>This class is deprecated</strong> use {@code fish.payara.ejb.http.client.RemoteEJBContextFactory}
+ *
+ * @deprecated in favor of package {@code fish.payara.ejb.http.client}
  * @author Arjan Tijms
  * @since Payara 5.191
  */
 @Deprecated
-public class RemoteEJBContextFactory implements InitialContextFactory {
+public class RemoteEJBContextFactory extends fish.payara.ejb.http.client.RemoteEJBContextFactory {
```

Comment:
I first was confused by this change but now I wonder why we did not solve the repackaging like you did in the first place. Did you ask about this?

Reply:
No I didn't. I honestly don't think there are any real world clients of these packages, so I just minimized the code duplication.